ZendeskSDK Urban Airship Example App/AppDelegate.m

```diff
@@ -100,7 +96,7 @@ - (void)application:(UIApplication *)application didRegisterForRemoteNotificatio
 - (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler
 {
     
-    [ZDKPushUtil handlePush:[[NSDictionary alloc] initWithDictionary:pushUtilUserInfo]
+    [ZDKPushUtil handlePush:[[NSDictionary alloc] initWithDictionary:userInfo]
              forApplication:application
```

Comment:
yeah it's fine as it is now, it was:

NSMutableDictionary *pushUtilUserInfo = [[NSMutableDictionary alloc] initWithDictionary:userInfo];
[pushUtilUserInfo setObject:userInfo[@"tid"] forKey:@"zendesk_sdk_request_id"];

But this was basically brought into the SDK.